src/Export/VTK.jl

```diff
@@ -27,13 +27,21 @@ end
 """
 struct VTKGridFile{VTK <: WriteVTK.DatasetFile}
     vtk::VTK
+    cellnodes::Vector{UnitRange{Int}}
```

Comment:
Maybe it is easier to maintain if we use a `Union{Vector{UnitRange{Int}},Nothing}` here to  distinguish between cases when `cellnodes` is accidentally (or intentionally) 0. One example here would be for MPI simulations we can run into the weird scenario that some processes have empty local grids. This might not be the only scenario tho.